canonical-web-and-design/canonicalwebteam.discourse
# Add generated sitemap.xml

# Done

- Add autogenerated `sitemap.xml`

# QA

- Add the module to a site that uses docs/engage pages/tutorials and make sure that the sitemap.xml is generated well
-  eg. /docs/sitemap.xml
